example/cumulus-tf/orca.tf

```diff
@@ -13,7 +13,7 @@ locals {
 # ORCA Module
 module "orca" {
   aws_region = var.region
-  source = "https://github.com/nasa/cumulus-orca/releases/download/v9.0.4/cumulus-orca-terraform.zip"
+  source = "https://github.com/nasa/cumulus-orca/releases/download/v10.0.0/cumulus-orca-terraform.zip"
```

Comment:
So Aurora Serverless v2 still works with orca v9.0.4, right? Since the build works without your change.

Reply:
I believe so. The orca team said it's required but I don't agree. It's only required if you force SSL so maybe that's what they meant.